index.html

```diff
@@ -539,6 +539,10 @@ <h2><dfn>MediaDevices</dfn> Additions</h2>
                         </li>
                       </ol>
                     </li>
+                    <li>
+                      Set the <a data-cite="html/#last-activation-timestamp">last activation timestamp</a>
```

Comment:
as far as I can determine, at the moment, the only web spec that sets the last activation timestamp is the HTML spec itself as part of the [activation notification](https://html.spec.whatwg.org/multipage/interaction.html#activation-notification), and that algorithm is somewhat involved, taking into account nested navigables.

I suspect this will need further refining (unless we can simply invoke the "activation notification" algorithm as is)

Reply:
We will send an issue to HTML repo and will discuss with HTML folks

Reply:
An adjacent issue I have just filed is [here](https://github.com/whatwg/html/issues/11330), as per Youenn's and Jan-Ivar's request during the interim meeting. For the issue Dom raises wrt specifying activation, a separate issue might have to be filed.